packages/ui-color-picker/src/ColorPicker/index.tsx

```diff
@@ -591,14 +602,18 @@ class ColorPicker extends Component<ColorPickerProps, ColorPickerState> {
           themeOverride={{ padding: '' }}
           renderAfterInput={this.renderAfterInput()}
           renderBeforeInput={this.renderBeforeInput()}
+          inputContainerRef={this.setLabelHeight}
```

Comment:
Wouldn't it be better to save the ref on the class and calculate the height on the style instead of using state? It would be more consistent for with the codebase, for sure and would make more sense setting the ref on inputContainerRef, instead of calculating the label hight. Is there anything preventing us doing that way?